### PR TITLE
mgr/dashboard: stats=false not working when listing buckets

### PR DESCRIPTION
**Pull request by avanthakkar**
Fixes: https://tracker.ceph.com/issues/51154
Signed-off-by: Avan Thakkar <athakkar@redhat.com>

Before:
Screenshot from 2021-08-20 14-47-22

After:
Screenshot from 2021-08-20 14-40-01

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/tasks/mgr/dashboard/test_rgw.py [hunk 1]
```diff
@@ -202,6 +202,12 @@ def test_all(self):
             'tenant': JLeaf(str),
         }, allow_unknown=True))
 
+        # List all buckets names without stats.
+        data = self._get('/api/rgw/bucket?stats=false')
```
Comment (alfonsomthd) [inline qa/tasks/mgr/dashboard/test_rgw.py:206]:
```suggestion
        data = self._get('/api/rgw/bucket')
```

`stats` is False by default

File: qa/tasks/mgr/dashboard/test_rgw.py [hunk 2]
```diff
@@ -202,6 +202,12 @@ def test_all(self):
...
+        self.assertStatus(200)
+        self.assertEqual(len(data), 1)
+        self.assertIn('teuth-test-bucket', data)
```
Comment (alfonsomthd) [inline qa/tasks/mgr/dashboard/test_rgw.py:209]:
```suggestion
        self.assertEqual(data, ['teuth-test-bucket'])
```

**Review by alfonsomthd — CHANGES_REQUESTED** [review]

File: qa/tasks/mgr/dashboard/test_rgw.py (hunk 1 above)
Comment (avanthakkar) [reply qa/tasks/mgr/dashboard/test_rgw.py:206]: > `stats` is False by default

Yes, but this is already tested [above](https://github.com/ceph/ceph/pull/42865/files#diff-095a0da450fe86d821140fea33b38b4b7d92c7048549efc11346954c1b8f3144L186). I wrote this 1 more test just to check it with JS `false` value (as Python `False` is different from `'false'`) similar as tested with  `true`.

**Comment by avanthakkar** [comment]
jenkins test make check

**Review by pereman2 — APPROVED** [review]
tested locally and working fine.

**Review by alfonsomthd — APPROVED** [review]

**Pull request merged**